Makefile

```diff
@@ -168,6 +172,14 @@ cluster-templates: $(KUSTOMIZE) ## Generate cluster templates
 	$(KUSTOMIZE) build $(E2E_TEMPLATES_DIR)/cluster-template-ubuntu > $(E2E_OUT_DIR)/cluster-template-ubuntu.yaml
 	$(KUSTOMIZE) build $(E2E_TEMPLATES_DIR)/cluster-template-centos > $(E2E_OUT_DIR)/cluster-template-centos.yaml
 	$(KUSTOMIZE) build $(E2E_TEMPLATES_DIR)/cluster-template-upgrade-workload > $(E2E_OUT_DIR)/cluster-template-upgrade-workload.yaml
+	touch $(E2E_OUT_DIR)/clusterclass.yaml
```

Comment:
@p-strusiewiczsurmacki-mobica hey, will you have a time to address this comment soon? We can then try to run CI tests on this one.

Reply:
@furkatgofurov7 Sorry, I've had to omit the notification email for this comment, and I've only seen it now.

I think that this should stay here. `clusterclass-templates` will not be used if non-clusterclass e2e test is being run, and `clusterclass.yaml` file is required by [e2e config](https://github.com/p-strusiewiczsurmacki-mobica/cluster-api-provider-metal3/blob/clusterclass-support/test/e2e/config/e2e_conf.yaml#L106). If it will not exist, the test will fail.
I had no better idea on how to fix that, but to create empty `clusterclass.yaml` file.

Reply:
ah right, now I got the point. Ideally, we could test the feature (simple cluster creation/destroy using CC would have been enough) as part of the current e2e tests setup, instead of having a separate Makefile target which is not part of it right now. 
@lentzi90 WDYT of the current approach?

Reply:
I think we can live with it, but I agree that making it part of the current e2e tests would be preferable. Enabling cluster class support in the controllers should be fine for all tests. They will anyway not use a ClusterClass unless defined in the test :slightly_smiling_face: